packages/manager/apps/dedicated/client/app/dedicatedCloud/service-pack/upgrade/steps/selection/selection.html

```diff
@@ -33,9 +33,15 @@ <h3 data-ng-bind="$ctrl.header"></h3>
         data-navigation-type="next"
     >
         <span
-            data-ng-if="$ctrl.stepper.getStep({ relativeIndex: 1 }).name === 'placeOrder'"
+            data-ng-if="$ctrl.stepper.getStep({ relativeIndex: 1 }).name === 'placeOrder'
+            && !$ctrl.isDowngrade"
             data-translate="dedicatedCloudDashboardTilesOptionsOrderSelection_checkoutButton"
         ></span>
+        <span
+            data-ng-if="$ctrl.stepper.getStep({ relativeIndex: 1 }).name === 'placeOrder'
```

Comment:
I guess it's copy/paste from above but I'm not too fond on checking something depending on some step name, if something should only be handled by a 'placeOrder' type it should be handled in the component handling that behaviour